libMBIN/Source/NMS/GameComponents/GcDestructableComponentData.cs

```diff
@@ -22,70 +22,72 @@ public class GcDestructableComponentData : NMSTemplate
         /* 0x028 */ public string TriggerAction;
         /* 0x038 */ public int IncreaseWanted;
         /* 0x03C */ public bool IncreaseFiendWanted;
-        /* 0x03D */ public bool NotifyEncounter;
-        [NMS(Size = 2, Ignore = true)]
-        /* 0x03E */ public byte[] Padding3E;
+        /* 0x040 */ public float IncreaseFiendWantedChance;
+        /* 0x044 */ public bool NotifyEncounter;
+        [NMS(Size = 3, Ignore = true)]
+        /* 0x045 */ public byte[] Padding3E;
 
         [NMS(Size = 0x10)]
-        /* 0x040 */ public string LootReward;
+        /* 0x048 */ public string LootReward;
 
-        /* 0x050 */ public int LootRewardAmountMin;
-        /* 0x054 */ public int LootRewardAmountMax;
+        /* 0x058 */ public int LootRewardAmountMin;
+        /* 0x05C */ public int LootRewardAmountMax;
+        /* 0x060 */ public bool CanDestroyFromStoredInteraction;
 
-        /* 0x058 */ public List<GcSubstanceAmount> GivesSubstances;
+        /* 0x068 */ public List<GcSubstanceAmount> GivesSubstances;
 
-        /* 0x068 */ public GcStatsEnum StatsToTrack;
+        /* 0x078 */ public GcStatsEnum StatsToTrack;
         [NMS(Size = 0x4, Ignore = true)]
-        /* 0x06C */ public byte[] Padding6C;
+        /* 0x07C */ public byte[] Padding6C;
 
         [NMS(Size = 0x10)]
-        /* 0x070 */ public string GivesReward;
-        /* 0x080 */ public bool HideReward;
-        [NMS(Size = 0x7, Ignore = true)]
-        /* 0x081 */ public byte[] Padding81;
+        /* 0x080 */ public string GivesReward;
+
+        /* 0x090 */ public bool RewardIfDestroyedByOther;
+        /* 0x091 */ public bool HideReward;
+        [NMS(Size = 0x6, Ignore = true)]
+        /* 0x092 */ public byte[] Padding81;
         [NMS(Size = 0x20)]
-        /* 0x088 */ public string OverrideRewardLoc;
-		/* 0x0A8 */ public float HardModeSubstanceMultiplier;
+        /* 0x098 */ public string OverrideRewardLoc;
+        /* 0x0B8 */ public float HardModeSubstanceMultiplier;
 
         [NMS(Size = 0x4, Ignore = true)]
-        /* 0x0AC */ public byte[] PaddingAC;
+        /* 0x0BC */ public byte[] PaddingAC;
 
-        /* 0x0B0 */ public List<GcRewardMissionOverride> RewardOverrideTable;
+        /* 0x0C0 */ public List<GcRewardMissionOverride> RewardOverrideTable;
 
-        /* 0x0C0 */ public bool ActivateLocatorsFromRarity;
+        /* 0x0D0 */ public bool ActivateLocatorsFromRarity;
         [NMS(Size = 0x7, Ignore = true)]
-        /* 0x0C1 */ public byte[] PaddingC1;
+        /* 0x0D1 */ public byte[] PaddingC1;
```

Comment:
The field names should also be updated (ie. this should be `PaddingD1`

Reply:
Many Padding names should be updated

Reply:
GcStatEnum was updated by HG sometime ago!

Reply:
Do you mean GcStatsEnum?